.github/workflows/benchmarks.yml

```diff
@@ -256,13 +256,14 @@ jobs:
 
       # Notify result to Slack only on schedule (nightly run)
       - if: github.event_name == 'schedule'
-        uses: slackapi/slack-github-action@37ebaef184d7626c5f204ab8d3baff4262dd30f0 # v1.27.0
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        uses: slackapi/slack-github-action@485a9d42d3a73031f12ec201c457e2162c45d02d # v2.0.0
         with:
-          channel-id: "#apm-server"
+          method: chat.postMessage
```

Comment:
See https://github.com/elastic/oblt-actions/pull/164, that PR runs some e2e validations in the CI and they worked fine.